bin/check-node-version

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env node
 const requiredVersion = require( '../package' ).engines.node,
-	semver = require( process.env.SEMVER_GLOBAL_PATH );
+	semver = require( 'semver' );
```

Comment:
See https://github.com/Automattic/wp-calypso/pull/4002 which got rid of this particular wrinkle.